docs/redirects.yaml

```diff
@@ -0,0 +1,35 @@
+"en/latest/":
+    # Redirects in /latest/ introduced for 1.10.0 doc reorganization
+    "introduction/": {replace_key: ""}
+    "introduction/flocker-supports.html": {replace_key: "supported/"}
+    "install/": {replace_key: ""}
+    "config/": {replace_key: ""}
+    "control/": {replace_key: ""}
+    "concepts/": {replace_key: "flocker-features/"}
+    "labs/installer.html": {replace_key: "flocker-standalone/installer.html"}
+    "labs/volumes-cli.html": {replace_key: "flocker-features/flockerctl.html"}
+    "labs/volumes-gui.html": {hostname: "clusterhq.com", replace_key: "volumehub/"}
+    "labs/swarm.html": {replace_key: "docker-integration/"}
+    "labs/compose.html": {replace_key: "docker-integration/"}
+    "labs/mesosphere.html": {replace_key: "mesos-integration/"}
+    "labs/kubernetes.html": {replace_key: "kubernetes-integration/"}
+    "labs/flockercontainers.html": {replace_key: "supported/flockercontainers.html"}
+    "labs/docker-plugin.html": {replace_key: "docker-integration/"}
```

Comment:
Yes, but it wasn't previously. We have had error reports from users landing on this page -- the link must be out there somewhere. We should add `introduction/flocker-plugin.html` as a link to there as well though.